src/community.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Community & Support
+styles:
+ - /styles/community.css
+---
+
+# Community & Support
+
+There are a few ways you can get involved with Workbox
+and learn more about current and future plans for the
+project as well as help the community or get support.
+
+## Stackoverflow
+
+Stackoverflow is a great place to asking and answering
+questions on how to use Workbox.
+
+If you do raise a question please you use the
+[workboxjs](https://stackoverflow.com/questions/tagged/workboxjs)
```

Comment:
It might be useful for us (nothing urgent) to populate this tag with at least one self-answered question. Right now it has no content so the users might be unsure it is active.